src/main/java/com/vinyl/model/Token.java

```diff
@@ -0,0 +1,71 @@
+package com.vinyl.model;
+
+
+import java.time.LocalDate;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.GenericGenerator;
+
+
+@Entity
+public class Token {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
+	@GenericGenerator(name = "native", strategy = "native")
+	private Integer id;
+	
+	@Column(name="HASH")
```

Comment:
be consistent with column naming (either all caps or all lowercase

Reply:
fixed